Userland/Applications/Mail/main.cpp

```diff
@@ -45,6 +45,11 @@ int main(int argc, char** argv)
         return 1;
     }
 
+    if (unveil("/tmp/portal/launch", "rw") < 0) {
+        perror("unveil");
+        return 1;
+    }
+
```

Comment:
Why is this change in this commit? Are you fixing an issue with unveil errors? Does that relate to the functional change the commit states it makes?

Reply:
`Desktop::Launcher` requires this.

Reply:
Yes, it's required by `Desktop::Launcher`.

Reply:
Then do that as a separate fix beforehand, or specify why this change is done in the commit message.

Reply:
> Then do that as a separate fix beforehand

It's not a fix. It belongs in the same commit and IMO is obvious enough to not require further details.